src/main/perl/MD.pm

```diff
@@ -55,10 +56,11 @@ sub _initialize
 
     my $st = $config->getElement($path)->getTree;
     $path=~m!/([^/]+)$!;
-    $self->{devname} = $1;
+    $self->{devname} = unescape($1);
```

Comment:
How are these devnames populated? i.e. Does this make the change backwards-incompatible?